AckooSDK/Classes/Networking/LocalError.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  LocalError.swift
+//  AckooSDK
+//
+//  Created by Sally Ahmed1 on 11/23/20.
+//
+
+import Foundation
+enum LocalError: Error {
+    case noInternetConnection
+    case parseResponseError
+    case parseErrorFailed
+    case unknown
+    case noData
+}
+
+extension LocalError {
+    var message: String {
+        switch self {
+        case .noInternetConnection:
+            return NSLocalizedString("localError.noInternet.message", comment: "")
```

Comment:
I wonder what NSLocalizedString is useful for ?

Reply:
localization for display depends on device language